### ISSUE TITLE
User can End Processes in Task Manager

### ISSUE THREAD

**Issue by jc6github** [lithnet/idle-logoff#9]
Standard users have discovered they can open the Task Manager and stop the Lithnet process rendering the app useless when they share this secret with others.  We cant limit access to the Task Manger as users need it to kill unresponsive programs.  This is on Win 10.

**Comment by ryannewington** [comment]
Hmm, I don't have a good answer for this one I'm afraid. The process must run in the user context to track the idle state, and users can always terminate their own processes. The only thing I can think of is a scheduled task the checks to see if the app is running, and restarts it if it is not.

**Comment by jc6github** [comment]
Thank you...I was attempting that...I notice if you stop the process and try to run the exe it again it prompts for Settings again but does not restart and getting error cant write to registry.  If I delete the location in the reg it saves ok.  Is there a way to restart the process with command line switch that bypasses  manually entering the settings?

**Comment by ryannewington** [comment]
The switch to use is '/start'

**Comment by jc6github** [comment]
Thanks Ryan... I think I have this working with Task Scheduler and a power shell script that checks if the process is running and if not starts it.  I used a trigger of "On Workstation Lock" as it is already set in group policy with screen saver.  I tried with On Idle but seemed to run into issues with monitors not coming out of power save mode.  

Here's the PS script if anyone else runs into this: 
$path = "C:\Program Files (x86)\Lithnet\Lithnet.IdleLogoff\lithnet.idlelogoff.exe"
get-process | where-object {$_.Path -eq $path } | measure-object | where-object { $_.Count -eq 0} | foreach-object {start-process "C:\Program Files (x86)\Lithnet\Lithnet.IdleLogoff\lithnet.idlelogoff.exe"/start }

Here's a helpful link for doing Task Scheduler with PS inGPO:
https://www.andersrodland.com/powershell-script-with-arguments-as-a-scheduled-task/
